Userland/Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -179,6 +179,17 @@ void PaintableBox::paint_border(PaintContext& context) const
         .bottom = computed_values().border_bottom(),
         .left = computed_values().border_left(),
     };
+
+    if (computed_values().clip().is_rect() && computed_values().position() == CSS::Position::Absolute) {
+        auto clip_rect = Gfx::IntRect {
+            computed_values().clip().to_rect().left_edge.to_px(Paintable::layout_node()),
+            computed_values().clip().to_rect().top_edge.to_px(Paintable::layout_node()),
+            computed_values().clip().to_rect().right_edge.to_px(Paintable::layout_node()) - computed_values().clip().to_rect().left_edge.to_px(Paintable::layout_node()),
+            computed_values().clip().to_rect().bottom_edge.to_px(Paintable::layout_node()) - computed_values().clip().to_rect().top_edge.to_px(Paintable::layout_node())
```

Comment:
I think you should factor out EdgeRect -> FloatRect to a helper (maybe `edge_rect.resolved(LayoutNode&)`). 
Then to get an IntRect you just do `edge_rect.resolved().to_rounded<int>()` here.